src/Bicep.Core/Syntax/SyntaxFactory.cs

```diff
@@ -200,6 +201,9 @@ public static Token CreateStringInterpolationToken(bool isStart, bool isEnd, str
             };
         }
 
+        public static FunctionCallSyntax CreateFunctionCall(string functionName)
+            => CreateFunctionCall(functionName, new FunctionArgumentSyntax[0]);
```

Comment:
Is this method necessary? `CreateFunctionCall(string functionName, params SyntaxBase[] argumentExpressions)` should allow a 0 length array

Reply:
[edit] never mind, I guess it's to disambiguate between the new overload and old.